### ISSUE TITLE
Cannot navigate away from "Keyboard Navigation" example

### ISSUE THREAD

**Issue by phryneas** [diogofcunha/react-virtualized-tree#107]
When on the ["Keyboard Navigation" example](https://diogofcunha.github.io/react-virtualized-tree/#/examples/keyboard-nav), none of the links to navigate away work any more.
Tested this in Firefox & Chrome.

**Comment by diogofcunha** [comment]
Hi @phryneas, thanks.

That's is actually intentional in that example, it uses focus trap in the tree and it can be deactivated by pressing escape. 

Could change to not trap focus on mount and do it for example on a checkbox toggle, will accept PR.

**Comment by phryneas** [comment]
Trapping the keyboard is all nice, but this traps mouse as well. It renders your documentation useless, as for someone not knowing about the escape key, if they want to go to another example, all they can do is close the tab.
